plans/tier1.fmf

```diff
@@ -300,3 +300,15 @@ adjust+:
         discover+:
             test+<:
                 - proxy-conversion/proxy_conversion
+
+    /error-after-ponr:
+        adjust+:
+          - enabled: false
+            when: distro == centos-8
+            because: On CentOS-8 the test doesn't behave well. The conversion does not report failure.
```

Comment:
```suggestion
    /error-after-ponr:
        enabled: false
        adjust+:
          - enabled: true
            when: distro == alma-9, rocky-9, oracle-9
            because: It will behave the same anywhere. Too expensive to run it on more hosts.
```
I suggest just limiting the test targets. Looks good otherwise.